scipy/spatial/kdtree.py

```diff
@@ -199,6 +199,10 @@ class KDTree(object):
             >>> import sys
             >>> sys.setrecursionlimit(10000)
 
+    See Also
+    --------
+    cKDTree : Implementation of `KDTree` in Cython
```

Comment:
Cython is just used to produce the Python interface. The implementation is in pure C++. (I know this because I wrote it.)